tests/software/pairtools/parse/main.nf

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PAIRTOOLS_PARSE } from '../../../../software/pairtools/parse/main.nf' addParams( options: ['suffix':'.raw'] )
+
+workflow test_pairtools_parse {
+
+    input = [ [ id:'test', single_end:false ], // meta map
+              file("https://raw.githubusercontent.com/open2c/pairtools/master/tests/data/mock.sam", checkIfExists: true) ]
```

Comment:
I have just realised that the module can take BAM input too. Can we not use the test data we have already to get the BAM file like [here](https://github.com/nf-core/modules/blob/5e18e36895ba7a63e864faad4105e14597ffcddb/tests/software/bedtools/genomecov/main.nf#L9) and the sizes file like [here](https://github.com/nf-core/modules/blob/5e18e36895ba7a63e864faad4105e14597ffcddb/tests/software/bedtools/complement/main.nf#L11)

Reply:
I will merge all of these in now but be great if you are able to update them all at some point if possible.

Reply:
I will do full test for the bam file. Once I confirmed the BAM file can work correctly, I will update the tool.